testsuite/MDAnalysisTests/transformations/test_fit.py

```diff
@@ -0,0 +1,228 @@
+    assert_array_almost_equal(test_u.trajectory.ts.positions, ref_u.trajectory.ts.positions, decimal=3)
+    
+
+def test_alignto_transformations_api(test_universe):
+    test_u = test_universe[0]
+    ref_u = test_universe[1]
+    ref_com = ref_u.atoms.center(None)
+    ref_u.trajectory.ts.positions -= ref_com
+    R = rotation_matrix(np.pi/3, [1,0,0])[:3,:3]
+    ref_u.trajectory.ts.positions = np.dot(ref_u.trajectory.ts.positions, R)
+    ref_u.trajectory.ts.positions += ref_com
+    transform = alignto(test_u, ref_u, weights=None)
+    test_u.trajectory.add_transformations(transform)
+    assert_array_almost_equal(test_u.trajectory.ts.positions, ref_u.trajectory.ts.positions, decimal=3)
+
+
+def test_fit_translation_bad_ag(test_universe):
+    ts = test_universe[0].trajectory.ts
+    test_u = test_universe[0]
+    ref_u  = test_universe[1]
+    # what happens if something other than an AtomGroup or Universe is given?
+    bad_ag = 1
+    with pytest.raises(ValueError): 
+        fit_translation(bad_ag, ref_u)(ts)
+
+
+def test_fit_translation_bad_center(test_universe):
+    ts = test_universe[0].trajectory.ts
+    test_u = test_universe[0]
+    ref_u  = test_universe[1]
+    # what happens if a bad string for center is given?
+    bad_center = "123"
+    with pytest.raises(ValueError): 
+        fit_translation(test_u, ref_u, center_of = bad_center)(ts)
+
+
+def test_fit_translation_bad_plane(test_universe):
+    ts = test_universe[0].trajectory.ts
+    test_u = test_universe[0]
+    ref_u  = test_universe[1]
+    # what happens if a bad string for center is given?
+    bad_plane = "123"
+    with pytest.raises(ValueError): 
+        fit_translation(test_u, ref_u, plane=bad_plane)(ts)
+
+
+def test_fit_translation_no_masses(test_universe):
+    ts = test_universe[0].trajectory.ts
+    test_u = test_universe[0]
+    # create a universe without masses
+    ref_u  = make_Universe()
+    # what happens Universe without masses is given?
+    with pytest.raises(AttributeError): 
+        fit_translation(test_u, ref_u, center_of="mass")(ts)
+
+
+def test_fit_translation_no_options(test_universe):
+    test_u = test_universe[0]
+    ref_u = test_universe[1]
+    fit_translation(test_u, ref_u)(test_u.trajectory.ts)
+    # what happens when no options are passed?
+    assert_array_almost_equal(test_u.trajectory.ts.positions, ref_u.trajectory.ts.positions, decimal=6)
+
+
+def test_fit_translation_com(test_universe):
+    test_u = test_universe[0]
+    ref_u = test_universe[1]
+    fit_translation(test_u, ref_u, center_of="mass")(test_u.trajectory.ts)
+    # what happens when the center o mass is used?
+    assert_array_almost_equal(test_u.trajectory.ts.positions, ref_u.trajectory.ts.positions, decimal=6)
+
+
+def test_fit_translation_plane(test_universe):
+    test_u = test_universe[0]
+    ref_u = test_universe[1]
+    # translate the test universe on the x and y coordinates only
+    fit_translation(test_u, ref_u, plane="xy")(test_u.trajectory.ts)
```

Comment:
Test all the planes.

Reply:
`test_fit_translation_plane()` calls `fit_translation` for all supported planes in the current PR code, so I think I can safely mark this comment as resolved.